contracts/modules/cover/Cover.sol

```diff
@@ -133,35 +133,29 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
 
     AllocationRequest memory allocationRequest;
     {
+
       if (_products.length <= params.productId) {
         revert ProductDoesntExist();
       }
 
       Product memory product = _products[params.productId];
 
-      if (product.isDeprecated) {
-        revert ProductDeprecated();
-      }
+       if (product.isDeprecated) {
+         revert ProductDeprecated();
+       }
```

Comment:
What exactly do you think you're doing here? :sweat_smile:

Reply:
They were commented out, and when I brought them back this happened :/